bin/gen_peptides.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+
+import sys
+import argparse
+import csv
+import pandas as pd
+from Bio import SeqIO
+from collections import Counter
+
+
+parser = argparse.ArgumentParser("Preprocessing of epitopes.")
+parser.add_argument('-i', '--input', metavar='FILE', type=str, help = 'TSV filename, containing paths to fasta files.')
+parser.add_argument('-o', '--output', metavar='FILE', type=argparse.FileType('w'), help='Output file: peptides.')
+parser.add_argument('-min', '--min_length', metavar='N', type=int, help='Min. length of peptides that will be generated.')
+parser.add_argument('-max', '--max_length', metavar='N', type=int, help='Max. length of peptides that will be generated.')
+args = parser.parse_args()
+
+
+# for each fasta file get proteins: Id and sequence
```

Comment:
Maybe we should use here the functionality already provided by FRED2: https://github.com/FRED-2/Fred2/blob/master/Fred2/IO/FileReader.py